neural_speed/convert/convert_stablelm.py

```diff
@@ -20,14 +20,15 @@
 # This script is similar to "convert-pt-to-ne.py"
 #
 import os
+import sys
 import struct
 import numpy as np
 from pathlib import Path
 import argparse
 from typing import (IO, TYPE_CHECKING, Any, Callable, Dict, Iterable, List, Literal, Optional, Sequence, Tuple, TypeVar,
                     Union)
 from transformers import AutoModelForCausalLM, AutoTokenizer
-import gguf
+import torch
```

Comment:
why we remove the support of gguf ?

Reply:
gguf format was generating gibberish text anyway with previous stablelm models, so I removed it and kept only ne format.